en/docs/integrate/develop/creating-artifacts/data-services/creating-data-services.md

```diff
@@ -17,7 +17,7 @@ Follow the steps given below to create the data service file:
 1.  Right-click the **Data Service Config** module in the project
     explorer and go to **New -> Data Service**. 
 
-    <img src="{{base_path}}/assets/img/integrate/tutorials/data_services/new-data-service.png" width="500"> 
+    <img src="{{base_path}}/assets/img/integrate/tutorials/data_services/new-data-service.png" width="700"> 
```

Comment:
please make the image links zoomable 
<a href="path"><img src="image-path"></a>